<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>窗口(&amp;W）</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="action_CloseAll"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="action_Cascase"/>
    <addaction name="separator"/>
    <addaction name="action_Previous"/>
    <addaction name="action_Next"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Next"/>
   <addaction name="action_Previous"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/document-new.png</normaloff>:/ICON/ICON/64x64/document-new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/folder-open.png</normaloff>:/ICON/ICON/64x64/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/save.png</normaloff>:/ICON/ICON/64x64/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/screenshot.png</normaloff>:/ICON/ICON/64x64/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/logout.png</normaloff>:/ICON/ICON/64x64/logout.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/arrow-1-backward.png</normaloff>:/ICON/ICON/64x64/arrow-1-backward.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/arrow-1-forward.png</normaloff>:/ICON/ICON/64x64/arrow-1-forward.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/cut.png</normaloff>:/ICON/ICON/64x64/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/copy.png</normaloff>:/ICON/ICON/64x64/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/paste.png</normaloff>:/ICON/ICON/64x64/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/door-closed.png</normaloff>:/ICON/ICON/64x64/door-closed.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/padlock-closed.png</normaloff>:/ICON/ICON/64x64/padlock-closed.png</iconset>
   </property>
   <property name="text">
    <string>关闭所有窗口(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关闭所有窗口</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/window-tile-horizontally.png</normaloff>:/ICON/ICON/64x64/window-tile-horizontally.png</iconset>
   </property>
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>平铺</string>
   </property>
  </action>
  <action name="action_Cascase">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/MD-photo-alt 2.png</normaloff>:/ICON/ICON/64x64/MD-photo-alt 2.png</iconset>
   </property>
   <property name="text">
    <string>折叠(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>折叠</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/arrow-4-right.png</normaloff>:/ICON/ICON/64x64/arrow-4-right.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>下一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/arrow-4-left.png</normaloff>:/ICON/ICON/64x64/arrow-4-left.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>前一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/about.png</normaloff>:/ICON/ICON/64x64/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="ICON.qrc">
     <normaloff>:/ICON/ICON/64x64/user-alt-3.png</normaloff>:/ICON/ICON/64x64/user-alt-3.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ICON.qrc"/>
 </resources>
 <connections/>
</ui>
